pages/History.qml

```diff
@@ -1454,8 +1451,12 @@ Rectangle {
         // limit results as per tx_max (root.txMax)
         var txs = root.txData.slice(tx_offset, tx_offset + tx_max);
 
+        // auto collapse if there is a single result
+        if(root.txPage === 1 && txs.length === 1)
+            auto_collapse = true;
+
         // make first result on the first page collapsed by default
-        if(auto_collapse && root.txPage === 1 && txs.length > 0 && (root.sortSearchString == null || root.sortSearchString === ""))
+        if(auto_collapse && root.txPage === 1 && txs.length > 0)
```

Comment:
Looks like we can drop `auto_collapse` entirely and use `root.txPage === 1 && txs.length === 1` instead.